test/e2e/app-dir/next-after-app/index.test.ts

```diff
@@ -200,25 +182,30 @@ describe.each(runtimes)('unstable_after() in %s runtime', (runtimeValue) => {
       })
 
       try {
-        const pendingReq = next.fetch('/delay', {
-          headers: { 'Next-Test-Proxy-Port': String(proxyServer.port) },
-        })
-
-        pendingReq.then(
-          async (res) => {
-            if (res.status !== 200) {
-              const msg = `Got non-200 response (${res.status}), aborting`
-              console.error(msg + '\n', await res.text())
-              abort(new Error(msg))
+        const pendingReq = next
+          .fetch(pathPrefix + '/delay', {
+            headers: { 'Next-Test-Proxy-Port': String(proxyServer.port) },
+          })
+          .then(
+            async (res) => {
+              if (res.status !== 200) {
+                const err = new Error(
+                  `Got non-200 response (${res.status}) for ${res.url}, aborting`
+                )
+                abort(err)
+                throw err
+              }
+              return res
+            },
+            (err) => {
+              abort(err)
+              throw err
             }
-          },
-          (err) => {
-            abort(err)
-          }
-        )
+          )
```

Comment:
this part isn't really relevant, just a minor improvement that helped catch crashes that'd previously fail silently (beware ye who try to do weird promise tricks)